### DISCUSSION TITLE
Snippets for RISC-V Assembly?

### DISCUSSION THREAD

**Discussion by evesdropper** [L3MON4D3/LuaSnip#673] [Q&A]
I tried writing RISC-V snippets using `assembly.lua` and yeah, that did not work.

![image](https://user-images.githubusercontent.com/82856360/204122714-ef4ae4ee-31a6-417c-a076-812f731721ef.png)

Do I need to declare some other language type, or is this not supported by Luasnip?

**Comment by L3MON4D3** [comment]
Ah, there's probably just some syntax-error in the file, we had bad error-messages for a bit (unfortunately that includes `v1.1.0`).
You could switch to master to find out where the error is coming from, the error is relayed properly there (also see #660)

**Reply by evesdropper** [reply]
Thanks! I updated to get better error messages and it turns out I added an extra comma somewhere. However, then I had some issues getting them to load with `.s` files - they don't show up at all, and no autosnippets are expanded.

Thought it might be an issue on my end, but then writing test snippets for markdown seemed to work fine, so I'm a little confused.

**Reply by L3MON4D3** [reply]
Mhmm you could also try the logging-branch for a bit, and set verbosity to info before loading any snippets (check doc-changes in #578), that should give some hints

**Reply by evesdropper** [reply]
Super late to this, but `:LuaSnipEdit` got the issue resolved - I was editing `assembly.lua` but the correct file should be `asm.lua`. Pity I can't write any snippets anymore though as I forgot all my assembly knowledge :rofl:
